### ISSUE TITLE
Problem regarding the targted attack

### ISSUE THREAD

**Issue by williamium3000** [Muzammal-Naseer/CDA#10]
Wonderful job!
I have a question regarding the targeted attack.
if the model (discriminator) is fixed.
loss = criterion(model(normalize(adv)), targte_label) + criterion(model(normalize(img)), label)
loss = criterion(model(normalize(adv)), targte_label)
these two losses have no difference since the second term has no effect on generator.

**Comment by Muzammal-Naseer** [comment]
Hi, 

Our loss functions are motivated by relativistic theory. This loss = criterion(model(normalize(adv)), targte_label) + criterion(model(normalize(img)), label) does provide a relativistic signal to the generator for maximized performance.

Thanks
